analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Sonar/EmptyNullableValueAccess.CSharp9.cs

```diff
@@ -125,3 +125,27 @@ public partial class Partial
         var v = nullable.Value;    // Noncompliant
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/6682
+public struct Repro_6682
+{
+    public bool SomeProperty { get; }
+
+    public void PatternMatching(Repro_6682? arg, bool condition)
+    {
+        if (condition)
+        {
+            arg = null;
+        }
+
+        if (arg is { SomeProperty: true })   // A null check
```

Comment:
Comments on lines containing `if` blocks come after 3 spaces from the last non-whitespace non-commenting char of the line, whereas comments on other lines come after just 2 spaces.
They are also not aligned with each other (which, in my understanding, would justify having more than 1 char between last non-whitespace non-commenting char and `//`).

Reply:
..sort-of aligned. The 1st one is aligned on tab distance, the rest is more to the left now as it would be just too far